youtube_dl/extractor/xnxx.py

```diff
@@ -6,14 +6,14 @@
 
 
 class XNXXIE(InfoExtractor):
-    _VALID_URL = r'^https?://(?:video|www)\.xnxx\.com/video(?P<id>[0-9]+)/(.*)'
+    _VALID_URL = r'^https?://(?:video|www)\.xnxx\.com/video-?(?P<id>[0-9a-z]+)/(.*)'
     _TEST = {
-        'url': 'http://video.xnxx.com/video1135332/lida_naked_funny_actress_5_',
+        'url': 'http://www.xnxx.com/video-43s6xfe/javso.com_-_beautiful_body_asian_softcore_idol',
```

Comment:
@TRox1972 Are you sure you haven't mistaken this line with `_VALID_URL` which is regex indeed?